bcda/auth/ssas.go

```diff
@@ -171,7 +174,7 @@ func (s SSASPlugin) AuthorizeAccess(tokenString string) error {
 	if !ok {
 		return errors.New("invalid ssas claims")
 	}
-	if _, err = adFromClaims(s.repository, claims); err != nil {
+	if _, err = GetProvider().GetAuthDataFromClaims(claims); err != nil {
```

Comment:
This creates an odd kind of circular dependency. Since it's already in a method of `SSASPlugin` that should just be referenced

```suggestion
	if _, err = s.GetAuthDataFromClaims(claims); err != nil {
```

Reply:
thanks, updating